modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -2,22 +2,37 @@
 //
 // * installing/installing_bare_metal/installing-bare-metal.adoc
 // * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
-// * installing_bare_metal/installing-bare-metal-network-customizations.adoc
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
 
 [id="installation-user-infra-machines-static-network_{context}"]
 = Configure advanced networking
 
-If you install {op-system-first} from an ISO image, you can add kernel arguments
-when you boot that image to configure the node's networking.
-The following table describes and illustrates how to use those kernel arguments.
+This section illustrates the networking configuration options that allow you to modify the install process. The following tables describe the kernel arguments you can use with:
 
-.Configure advanced networking
+* The {op-system} live installer boot prompt by using routing and bonding for ISO installs.
+
+* The {op-system} live installer command by using `core.inst` for ISO or PXE installs.
+
+* The `coreos-installer` command for ISO installs.
+
+[IMPORTANT]
+====
+When using networking kernel arguments, you must specify `rd.neednet=1`.
```

Comment:
This was a comment from Benjamin Gilbert. I don't know if this makes sense as a note, or if it should be included in each table. Also, is this required for each install type (boot prompt, `core.inst`, and `coreos-installer`?